Update Astro Alerts (#2344)
astronomer/docs: astro/alerts.md

  If you do not have permission to install apps into your Slack workspace, you can still create the app, but you will need to request that an administrator from your team completes the installation.
  :::

  ![Create your app in a Slack workspace](/img/docs/slack_alerts_1.3.png)

4. Select **Incoming webhooks**.

5. On the **Incoming webhooks** page, click to turn on **Activate Incoming Webhooks**.

>   ![Activate an incoming webhook](/img/docs/slack_alerts_1.5.png)

Reviewer: That is a good point! 👍

     You can add more DAGs or tasks after you create your alert. 

8. Click **Create alert**.

## Step 3 (Optional): Test your DAG run failure alert

Because manually stopping your DAG triggers Astro to send an alert, you can test your integration by manually testing your DAG.

1. To test your alert, **Open Airflow** for the Deployment where you configured your alert.

2. Choose the DAG that has your alert configured.

3. Trigger a DAG run.

> 4. Manually mark your DAG's state as **Failed** by clicking the DAG run in the Grid view, then clicking **Mark state as...** and choose **Failed**.

Reviewer 2: ```suggestion
4. Manually mark your DAG's state as **Failed** by clicking the DAG run in the Grid view, then clicking **Mark state as...** > **Failed**.
```

     You can add more DAGs or tasks after you create your alert. 

8. Click **Create alert**.

> ## Step 3 (Optional): Test your DAG run failure alert

Reviewer 2: ```suggestion
## Step 3: (Optional) Test your DAG run failure alert
```
This seems slightly more consistent with our optional styling in steps

> Astro alerts work whether your DAG run is manual or scheduled. So, you can easily test your configures Astro alert by failing your DAG manually.

Reviewer 2: ```suggestion
Astro alerts work whether your DAG run is manual or scheduled, so you can test your configured Astro alerts by failing your DAG manually.
```

> 1.  In the Cloud UI, open the Deployment where you configured your alert and click **Open Airflow**.

Reviewer 2: ```suggestion
1. In the Cloud UI, open the Deployment where you configured your alert and click **Open Airflow**.
```

> 2. Choose the DAG that has your alert configured.

Reviewer 2: I think we want to start driving usage of the DAGs tab in the Cloud UI, which is basically a reskinned Airflow UI that's more integrated with Astro features. These steps look great, but can you tweak the wording and screenshots to use the DAGs page workflow instead?

![Screenshot 2023-08-16 at 11 55 48 AM](https://github.com/astronomer/docs/assets/74574233/d0ac3f92-3814-484b-a947-05682274797f)
